src/component/plasmic/shared/PlasmicActionBar.module.css

```diff
@@ -718,7 +717,7 @@
   > picture
   > img {
   margin-left: 10px;
-  margin-top: 10px;
+  margin-bottom: 5px;
```

Comment:
This is fine for now, but next time we run `plasmic sync` it will be lost. Let's make the same change in Plasmic.